csm/MotorMan.py

```diff
@@ -0,0 +1,25 @@
+class MotorMan():
+
+    def __init__(self):
+        self.orientationX = 0
+        self.orientationY = 0
+
+    def opticalFlowToMotorDeltas(self, optical_flow):
+        """
+        Converts optical flow data (vectors) into deltas motor must
+        turn to re-center camera view.
+        """
+        deltaX, deltaY = 0,0
+
+        pass
+        
+        return deltaX, deltaY
```

Comment:
I don't think these would have to be returned. Rather, they'd have to become signals, but I guess this is fine until Thor fills it out

Reply:
Good point, I'm fine with sticking simple unless needed otherwise